README.md

```diff
@@ -38,6 +38,33 @@ spec:
 
 This will discover deployments/daemonsets/statefulset automatically where `foo-configmap` or `foo-secret` is being used either via environment variable or from volume mount. And it will perform rolling upgrade on related pods when `foo-configmap` or `foo-secret`are updated.
 
+You can restrict this discovery to only `ConfigMap` or `Secret` objects that
+are tagged with a special annotation. To take advantage of that, annotate
+your deployment/daemonset/statefulset like this:
+
+```yaml
+kind: Deployment
+metadata:
+  annotations:
+    reloader.stakater.com/search: "true"
+spec:
+  template:
+```
+
+and Reloader will trigger the rolling upgrade when any `ConfigMap` or `Secret` annotated like this:
+and then annotate your `ConfigMap` or `Secret` so:
+
+```yaml
+kind: ConfigMap
+metadata:
+  annotations:
+    reloader.stakater.com/match: "true"
+data:
+  key: value
+```
+
+is modified, provided it is being used in an environment variable or a volume mount.
```

Comment:
Can you please remove this, 
> and then annotate your `ConfigMap` or `Secret` so:

and this

> is modified,

Also, can you please add a line that these two annotations `reloader.stakater.com/search: "true"`  and`reloader.stakater.com/auto: "true"` doesn't work together. If you have a `reloader.stakater.com/auto: "true"` annotation in your deployment then it will always reload the pod upon a change in configmap and secret, whether you have this `reloader.stakater.com/match: "true"` annotation or not.

Reply:
Done!